servers/republik/modules/crowdfundings/lib/Mail.js

```diff
@@ -522,24 +523,23 @@ mail.getPledgeMergeVars = async (
   })
 
   pledgeOptions
-    // Sort by packageOption.order in an ascending manner
-    .sort(
-      (a, b) =>
-        a.packageOption &&
-        b.packageOption &&
-        a.packageOption.order > b.packageOption.order ? 1 : 0
-    )
+    // Sort by price
+    .sort((a, b) => descending(a.price, b.price))
     // Sort by sequenceNumber in an ascending manner
-    .sort(
-      (a, b) =>
-        a.membership &&
-        b.membership &&
-        a.membership.sequenceNumber < b.membership.sequenceNumber ? 1 : 0
-    )
+    .sort((a, b) => ascending(
+      a.membership && a.membership.sequenceNumber,
+      b.membership && b.membership.sequenceNumber
+    ))
     // Sort by userID, own ones up top.
-    .sort(
-      (a, b) => a.membership && a.membership.userId !== pledge.userId ? 1 : 0
-    )
+    .sort((a, b) => descending(
+      a.membership && a.membership.userId === user.id,
+      b.membership && b.membership.userId === user.id
+    ))
+    // Sort by packageOption.order in an ascending manner
+    .sort((a, b) => ascending(
+      a.packageOption && a.packageOption.order,
+      b.packageOption && b.packageOption.order
+    ))
```

Comment:
IMO would be clearer if in one sort (and slightly faster). Following is possible because `ascending` and `descending` return `0` if the same:

```js
    .sort((a, b) => (
      ascending(
        a.packageOption && a.packageOption.order,
        b.packageOption && b.packageOption.order
      ) ||
      descending(
        a.membership && a.membership.userId === user.id,
        b.membership && b.membership.userId === user.id
      ) ||
      ...
    )
```